lib/hooks/useTypography/index.treat.ts

```diff
@@ -0,0 +1,87 @@
+import { style, css } from 'sku/treat';
```

Comment:
I reckon we should call this `typography.treat.ts`?

Reply:
Done. Also renamed the directory to better reflect where we landed